example/src/App.js

```diff
@@ -49,49 +49,58 @@ class App extends Component {
             {dragState => (
               <div
                 style={{
-                  position: 'fixed',
+                  position: "fixed",
                   left: dragState.x - 25,
                   top: dragState.y - 25,
                   width: 50,
                   height: 50,
-                  background: 'lime',
-                  pointerEvents: 'none'
+                  background: "lime",
+                  pointerEvents: "none"
                 }}
               />
             )}
           </DragComponent>
-          <Droppable accepts={['red', 'blue']}>
-            {dragState => (
-              <div
-                {...dragState.events}
-                style={{
-                  background:
-                    dragState.type === 'red' && dragState.isOver
-                      ? 'red'
-                      : 'blue',
-                  width: 200,
-                  height: 200
-                }}
-              >
-                isOver: {dragState.isOver ? 'true' : 'false'}
-                <br />
-                accepts:{' '}
-                {dragState.type === 'red' && dragState.isOver
-                  ? 'true'
-                  : 'false'}
-              </div>
-            )}
+          <Droppable
+            id="redDrop"
+            accepts={["red", "blue"]}
+            subscribeTo={["isOver", "type"]}
+          >
+            {dragState => {
+              console.log("I rendered")
+              return (
+                <div
+                  {...dragState.events}
+                  style={{
+                    background:
+                      dragState.type === "red" && dragState.isOver
+                        ? "red"
+                        : "blue",
+                    width: 200,
+                    height: 200
+                  }}
+                >
+                  isOver: {dragState.isOver ? "true" : "false"}
+                  <br />
+                  accepts:{" "}
+                  {dragState.type === "red" && dragState.isOver
+                    ? "true"
+                    : "false"}
+                  <br />
+                  I should only rerender if I'm hovered.
```

Comment:
I guess it's better to escape this string with `{""}` to fix syntax highlighting.

Reply:
Good catch, I wasn't sure what was throwing it off.